### PR TITLE
Send to Publish changes

### PR DESCRIPTION
**Pull request by elit0451**
Updating sendToPublish controller + making some tree arrows to appear when there are descendants.

### CODE REVIEW

File: src/Umbraco.Web/Trees/MacrosTreeController.cs [hunk 1]
```diff
@@ -19,6 +20,14 @@ namespace Umbraco.Web.Trees
     public class 
         MacrosTreeController : TreeController
     {
+        protected override TreeNode CreateRootNode(FormDataCollection queryStrings)
```
Comment (clausjensen) [inline src/Umbraco.Web/Trees/MacrosTreeController.cs]: I'm not really sure if this was supposed to be a part of this pull request?

File: src/Umbraco.Web/Trees/MemberGroupTreeController.cs [hunk 2]
```diff
@@ -19,5 +19,13 @@ protected override IEnumerable<TreeNode> GetTreeNodesFromService(string id, Form
                 .OrderBy(x => x.Name)
                 .Select(dt => CreateTreeNode(dt.Id.ToString(), id, queryStrings, dt.Name, "icon-item-arrangement", false));
         }
+
+        protected override TreeNode CreateRootNode(FormDataCollection queryStrings)
```
Comment (clausjensen) [inline src/Umbraco.Web/Trees/MemberGroupTreeController.cs]: same goes for this 😄

File: src/Umbraco.Web/Trees/MacrosTreeController.cs (hunk 1 above)
Comment (elit0451) [reply src/Umbraco.Web/Trees/MacrosTreeController.cs]: Moved to https://github.com/umbraco/Umbraco-CMS/pull/3398.

File: src/Umbraco.Web/Trees/MemberGroupTreeController.cs (hunk 2 above)
Comment (elit0451) [reply src/Umbraco.Web/Trees/MemberGroupTreeController.cs]: -> https://github.com/umbraco/Umbraco-CMS/pull/3398

**Pull request merged**